src/charts/donut.js

```diff
@@ -556,6 +558,24 @@ define(function(require) {
 
         // API
 
+        /**
+         * Gets or Sets the centeredTextFunction of the chart. If function is provided
+         * the format will be changed by the custom function's value format.
+         * The default format function value is "${d.percentage}% ${d.name}"
+         * The callback will provide the data object with id, name, percentage, and quantity
```

Comment:
I would add that we also provide whatever components the user adds in each data entry.

I would also add an example, like:
```
@example
 donutChart
     .centeredTextFunction((d) => `${d.id} ${d.quantity}`);
```